templates/build_matrix

```diff
@@ -1,4 +1,6 @@
 darwin amd64
+darwin amd64-v2
+darwin amd64-v3
```

Comment:
right, I'll remove them.

Since I'll have to run CI again anyway, I'll remove all v2 and v3 for fs-repo-* (see https://github.com/ipfs/distributions/pull/790#issuecomment-1346863565), the goal is to improve daemons like Kubo.